backend/.gitignore

```diff
@@ -5,6 +5,7 @@ bin/
 !gradle/wrapper/gradle-wrapper.jar
 !**/src/main/**
 !**/src/test/**
+/bin
```

Comment:
seems to me that this didn't work out. There are a lot of binary files in the PR